snapshot.sh

```diff
@@ -1,6 +1,22 @@
 #!/bin/bash -e
 
-function help_menu() {
+# Needs:
+# 1. A "parse_prereqs()" func that bails when lacking prereqs: e.g. aws cli, jq, cosmovisor.service
+
+# # # # #
+# Init vars:
+# CC: If these are set via parse_args(), why do we init them here with empy values?
```

Comment:
We probably don't, but that's just a habit that I've formed when working with getopts for some reason. They can most likely be deleted w/o issue.